operate/webapp/src/main/java/io/camunda/operate/Application.java

```diff
@@ -99,6 +99,8 @@ private static Map<String, Object> getWebProperties() {
     return Map.of(
         "server.servlet.session.cookie.name",
         "OPERATE-SESSION",
+        "spring.thymeleaf.check-template-location",
+        "true",
```

Comment:
❓ What is this doing? :D

Reply:
I am also not a thymeleaf expert. Since it is on the classpath, it was complaining that it couldn't find some an `index` when starting Zeebe. I disabled that in the `application.properties`, but then Operate was not able to serve the frontend.

It is similar to the `add-mappings` configuration which is for the time being, a "solution" to get it working, it must be reworked anyway when working on the single application.